### PR TITLE
Rule update more user known macros

### PR DESCRIPTION
**Pull request by kunfoo**
**What type of PR is this?**

/kind rule-update

**Any specific area of the project related to this PR?**

/area rules

**What this PR does / why we need it**:

we use user_known_\* macros extensively to adapt falco rules to our environment. however, certain rule conditions do not have such a macro.

```release-note
NONE
```

### CODE REVIEW

**Comment by mabushey** [comment]
This is exactly what I need to remove all the containerd `Modify Shell Configuration File` warnings.

File: rules/falco_rules.yaml [hunk 1]
```diff
@@ -1243,9 +1251,14 @@
...
 
+- macro: user_known_write_below_etc_conditions
+  condition: (never_true)
+
 - rule: Write below etc
   desc: an attempt to write to any file below /etc
-  condition: write_etc_common
+  condition: >
+    write_etc_common and
+    not user_known_write_below_etc_conditions
```
[the remark this answers is not in the document]
Comment (kunfoo) [reply rules/falco_rules.yaml:1261]: You are right, but the already existing macro is called `user_known_write_below_etc_activities`. I did not find this macro, as it is hidden in the very long definition of `write_etc_common`. I think it is better to include `user_known_write_below_etc_activities` directly in the rule's condition, as this is how it is done with all the other `user_*` macros in the rule conditions.

**Comment by jasondellaluce** [comment]
Hey @kunfoo, what this the current state of this? If we still want this, you may need to remove the merge commit and rebase your branch. That's a requirement for this repo.

**Review by Kaizhe — APPROVED** [review]

**Pull request closed without merging**